types/src/move_resource.rs

```diff
@@ -8,12 +8,12 @@ use anyhow::Result;
 pub trait MoveStorage {
     /// Returns a vector of Move resources as serialized byte array
     /// Order of resources returned matches the order of `access_path`
```

Comment:
Nit: comments need to be updated, too (and below).

Reply:
Updated ! 👍